pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/UpdateTemplatesInPlace.java

```diff
@@ -190,6 +190,8 @@ private void traverseEventConfigurations(Handle handle, long studyId) {
     }
 
     // Best-effort attempt at making a unique identifier for event configuration.
+    // watch out for scenarios where diff events might end up with same unique identifier
+    // ex:- trigger and order are same but diff pre/cancel expressions !
```

Comment:
If the `preconditionExpr` and other expressions result in technically different events, should we be incorporating those into the hash?

Reply:
Ahh- I see your note in the description.

Reply:
created tkt to track it https://broadinstitute.atlassian.net/browse/DDP-8575